PyInstaller/building/toc_conversion.py

```diff
@@ -132,7 +132,7 @@ def __collect_data_files_from_zip(zipfilename):
         # 'PyInstaller.config' cannot be imported as other top-level modules.
         from ..config import CONF
         workpath = os.path.join(CONF['workpath'], os.path.basename(zipfilename))
-        os.makedirs(workpath)
+        os.makedirs(workpath, exist_ok=True)
```

Comment:
I'm sorry, but this is Python-3-only :-(
So wrapping it in try/except, catching the IOError would be a more portable solution.